install_dependencies.R

```diff
@@ -1,5 +1,9 @@
 install.packages(c('testthat', 'vdiffr', 'spelling','covr','remotes'), repos = 'http://cran.us.r-project.org', type='win.binary')
-install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')
+download.file('https://github.com/Open-Systems-Pharmacology/rSharp/releases/download/v1.0.0/rSharp-v1.0.0-Windows-r_4.4.0.zip', destfile = "rsharp.zip", mode = "wb")
```

Comment:
Where is `install_depenendies` used? @Felixmil

Reply:
I am not sure it is used anywhere anymore.

- [ ] Delete `install_depenencies.R`